beeai/agents/rebase_agent.py

```diff
@@ -161,23 +169,35 @@ def prompt(self) -> str:
 
           5. Verify and adjust the changes:
               * Use `rpmlint` to validate your .spec file changes and fix any new errors it identifies.
-              * Generate the SRPM using `rpmbuild -bs` (ensure your .spec file and source files are correctly
-                copied to the build environment as required by the command).
+              * Generate the SRPM using `centpkg srpm --buildroot rhel-N`, where N is extracted from the cNs dist_git_branch.
+              * Ensure your .spec file and source files are correctly copied to the build environment as required by the command.
+              * Take the path to the SRPM file and build the RPM in Copr using `build_package` tool, confirm it succeeds.
+                IMPORTANT: 
+                    - if the build fails, due to a kerberos ticket issue, you must try to re-run the build_package tool at least 3 times and wait 10 seconds between each attempt.
```

Comment:
I would maybe rather try to codify this, if possible

Reply:
my thinking was that this really does look like a pseudocode and the operation (retry, with the retry interval, based on specific "exception") is deterministic, if we had a good way of displaying the tool calls, I would be still for trying to do this with tool

Reply:
I wonder if we can have the kerberos kinit process outside of agents. Something like a sidecar container that retries kinit every hour in openshift.

Reply:
> I wonder if we can have the kerberos kinit process outside of agents. Something like a sidecar container that retries kinit every hour in openshift.

It is outside of the agent, it is inside the mcp gateway/server. The agent is only retrying the call if the tool invokation, ends with a kerberos error. 
Do you mean something like having a `kinit`  command running every hour on a cron job inside the mcp gateway? I like the idea. Don't know what @nforro thinks (he also has to kinit for his tools)?

Reply:
> Do you mean something like having a kinit command running every hour on a cron job inside the mcp gateway?

Exactly. We could ask other teams how they do this.